components/Header.tsx

```diff
@@ -6,9 +6,9 @@ export default function Header() {
   const [isMenuOpen, setIsMenuOpen] = useState(false)
 
   return (
-    <header className="fixed w-full backdrop-blur-md top-0 z-50">
+    <header className="fixed w-full top-0 z-50">
```

Comment:
If we remove the blur the text of the header overlaps all the content of the website resulting in not very pleasent views like the one below. I would keep the blur or make it intranparent. I personally would keep it as it is (for me it looks ok on mobile). But am also ok with an intransparent one.

<img width="1491" alt="Screenshot 2024-12-16 at 08 25 36" src="https://github.com/user-attachments/assets/66ddc43b-853a-4134-80dd-21a4444d784e" />

Reply:
Than rather the non-transparent variant.